doc/gallery-src/framework/run_grids3_rzt.py

```diff
@@ -0,0 +1,35 @@
+"""
+Make a Theta-R-Z grid
+=====================
+
+This builds a 3-D grid in Theta-R-Z geometry by specifying the theta, r, and z 
+dimension bounds explicitly. 
+
+Learn more about :py:mod:`grids <armi.reactor.grids>`.
+"""
+import itertools
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+import numpy as np
+
+from armi.reactor import grids
+
+import armi
+
+armi.configure()
+
+fig = plt.figure()
+theta = np.linspace(0, 2 * np.pi, 10)
+rad = np.linspace(0, 10, 10)
+z = np.linspace(5, 25, 6)
+rz_grid = grids.ThetaRZGrid(bounds=(theta, rad, z))
+
+
+xyz = []
+for i, j, k in itertools.product(range(9), range(9), range(5)):
```

Comment:
For these ranges, can you just do `for i, j, k in itertools.product(range(len(theta) - 1), range(len(rad) - 1), range(len(z) - 1):`?

Reply:
yeah